pom.xml

```diff
@@ -75,16 +87,12 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.8.0</version>
         <configuration>
-          <compilerArgs>
-            <arg>-h</arg>
```

Comment:
What is the consequence of removing these compiler args?

Reply:
I don't think JNI header is used in this lib.
There should be no impact to remove them. I could also add them back to be safer.